src/App.js

```diff
@@ -18,7 +19,7 @@ function App() {
   const [token, setToken] = useState(null);
   const [repository, setRepository] = useState(null);
   const [show, setShow] = useState(true);
-  const isAuthorized = token && repository;
+  const isAuthorized = token && repository && true;
```

Comment:
I don't understand why you're doing this change.

Reply:
I had this warning from chromium browser

```
react_devtools_backend.js:4012 Warning: Failed prop type: Invalid prop `isAuthorized` of type `string` supplied to `NavigationBar`, expected `boolean`.
    at NavigationBar (http://localhost:3000/humphd/my-photohub/static/js/bundle.js:401:5)
    at App (http://localhost:3000/humphd/my-photohub/static/js/bundle.js:60:76)
```
I was able to remove this warning with this change.

Reply:
@batunpc, just a suggestion: to make value of `isAuthorized` more clear you could do something like:
`const isAuthorized = token !== null && repository !== null` It will also not raise the warning.

Reply:
@SerpentBytes, thanks for the suggestion! 
This warning will not appear anymore after adding the `const isAuthorized = token && repository && true;`

Reply:
```ts
const isAuthorized = !!(token && repository); // or Boolean(...) 
```